canonical/maas-ui
# feat(vlans): Include subnet_ids on VLAN type and use where valid lp#1959648

## Done

- Added `subnet_ids` to base VLAN type
- Included subnet_ids in vlan factory
- Use list of IDs to retrieve subnets from store (where it makes sense to do so)

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to subnets
- Click on a VLAN
- Ensure the page loads correctly and the correct VLANs are shown

**Note for reviewers: If there's anywhere else you think we could use this list of IDs, please let me know**

## Launchpad issue

lp#1959648

https://bugs.launchpad.net/maas-ui/+bug/1959648
